cbor.go

```diff
@@ -10,6 +11,14 @@ import (
 // from https://www.iana.org/assignments/cbor-tags/cbor-tags.xhtml#tags
 const SignMessageCBORTag = 98
 
+var signMessagePrefix = []byte{'\xd8', SignMessageCBORTag}
```

Comment:
Is the `0xd8` magic number documented anywhere? If so, might be nice to add a link to that.

Reply:
yeah good call I'll add a comment and link